src/patternfly/components/Label/label.scss

```diff
@@ -122,7 +122,9 @@
 
   // Content
   --pf-c-label__content--Color: var(--pf-global--Color--100);
+  --pf-c-label__content--MaxWidth: initial;
```

Comment:
Do you know if the `max-width` default for a flex item is `none`? That's what `initial` initial is for `max-width`.  https://developer.mozilla.org/en-US/docs/Web/CSS/max-width

I ask because `min-width` for a flex item is `min-content`, which is different from the `initial` value.

Reply:
I changed it to 100%, which is max-width for pf-flex children. Also that seems to make more sense in general, right?